backend_new/prisma/seed-helpers/unit-type-factory.ts

```diff
@@ -0,0 +1,15 @@
+import { Prisma } from '@prisma/client';
+
+export const unitTypeFactory = (i: number): Prisma.UnitTypesCreateInput => ({
+  ...unitTypeArray[i % unitTypeArray.length],
+});
+
+const unitTypeArray = [
+  { name: 'studio', numBedrooms: 0 },
+  { name: 'oneBdrm', numBedrooms: 1 },
+  { name: 'twoBdrm', numBedrooms: 2 },
+  { name: 'threeBdrm', numBedrooms: 3 },
+  { name: 'fourBdrm', numBedrooms: 4 },
+  { name: 'SRO', numBedrooms: 0 },
+  { name: 'fiveBdrm', numBedrooms: 5 },
```

Comment:
Do the unit type names have an enum? If not, recommend adding one and using that instead of strings.

Reply:
updated!